star_tides/services/databases/mongo/models/contact_model.py

```diff
@@ -0,0 +1,29 @@
+'''ContactModel representing Contact objects in the MongoDB database.'''
+
+from mongoengine import Document, StringField, EmailField, UUIDField, URLField
+from mongoengine.fields import EmbeddedDocumentField, ListField, EnumField
+from .location_model import LocationModel
+from .engagement_model import EngagementModel
+from .availability import Availability
+
+class ContactModel(Document):
+    '''Contact MongoDB model.'''
+
+    user_id = UUIDField(binary=False, required=False)
+    name = StringField(required=True)
+    email = EmailField(required=False)
+    phone_number = StringField(required=False)
+    job_title = StringField(required=False)
+    website_url = URLField(required=False)
+    location = EmbeddedDocumentField(LocationModel, required=True)
+    languages = ListField(required=False)
+    availability = EnumField(Availability, required=True)
+    engagement = EmbeddedDocumentField(EngagementModel, required=True)
+    statuses = ListField(required=False)
```

Comment:
Most of this would make sense in the `User` model. What is the purpose of `Contact` model?

Reply:
It represents someone in the KB who doesn't have write access but is part of the network. For example, this may be someone who doesn't work at GMU but has been involved in several STAR-TIDES projects.